models/replications.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+var PathReplication = "/replication/policies"
+
+type ReplicationBody struct {
+	Name        string `json:"name,omitempty"`
+	Description string `json:"description,omitempty"`
+	SrcRegistry struct {
+		ID int `json:"id,omitempty"`
+	} `json:"src_registry,omitempty"`
+	DestRegistry struct {
+		ID int `json:"id,omitempty"`
+	} `json:"dest_registry,omitempty"`
+	DestNamespace interface{} `json:"dest_namespace,omitempty"`
```

Comment:
Shouldn't that be a `string`and not an `interface{}` ?

Reply:
correct. I've changed this